providers/implementations/ciphers/cipher_aes_cbc_hmac_sha.c

```diff
@@ -16,6 +16,8 @@
 
 /* Dispatch functions for AES_CBC_HMAC_SHA ciphers */
 
+/* Only for SSL3_VERSION and TLS1_VERSION */
```

Comment:
I'd almost rather hardcode the literal constants for the versions (with a comment) than pull in `ssl.h` here...

Reply:
Hmmm. Really? I'm not sure that is better?